### DISCUSSION TITLE
Action buttons on listView and itemView

### DISCUSSION THREAD

**Discussion by zdllucky** [keystonejs/keystone#8650] [Feature requests]
In Keystone 5 there was a pretty useful feature - Custom action buttons for the admin panel. It is actually very handy when it comes to transform basic CRUD admin panel to more comprehensive one. I tried to lookup for it in code both in any PR or roadmap but did not find any mention. 

Could you please tell me is it somewhere in backlog and if not how to vote up for it? :)

**Comment by tv42** [comment]
See also https://github.com/keystonejs/keystone/discussions/5315

**Reply by zdllucky** [reply]
I think what they want somehow corresponds yet differs from what I mean. The future I request was implemented already in v5. And it was configured in admin section, not listConfigs one.
